plugins/notifications-frontend/src/components/NotificationsPage/NotificationsPage.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Page, RoutedTabs } from '@backstage/core-components';
+
+import { PersonalNotifications } from '../PersonalNotifications';
+import { SendNotification } from '../SendNotification';
+import { SystemNotifications } from '../SystemNotifications';
+
+export const NotificationsPage = () => {
+  const params = useParams();
+  const isSend = params['*'] === 'send';
+
+  const routes = [
+    {
+      path: 'personal',
+      title: 'Personal',
+      children: <PersonalNotifications />,
+    },
+    {
+      path: 'updates',
+      title: 'Updates',
+      children: <SystemNotifications />,
```

Comment:
What are system notifications? Is this like the `announcements` plugin that already exists?

Reply:
A system notification is without target user, delivered to all users. They can be listed and marked as read like the "personal" notifications but in addition they are presented as snackbars on any page the user currently is.

The same mechanism for sending messages is used, no matter the target audience are all users.

Reply:
So basically they are like the announcements plugin here https://github.com/procore-oss/backstage-plugin-announcements ?

Reply:
Thanks. It's worth of exploring whether the notifications can emit announcements instead of recent way of rendering the system notifications.

Do you know if there is a way for external systems to create an announcement?
If it is not or is not handy, the integration notifications to announcements is useful.

Reply:
There's an API to create announcements and the create is behind the permissions framework https://github.com/procore-oss/backstage-plugin-announcements/blob/main/plugins/announcements-common/src/permissions.ts#L3. So basically if you have no permissions in use it's possible. Otherwise, it's the same problem as you have with external caller JWT which probably should be handled by the backstage core instead of having each plugin do some magic headers to verify the sender.

Reply:
we provide our own system notifications. this way you get it via the same API and view it in the same UI. if it is not required then we can remove it. but why remove something so simple that seems to have a real use case?
one thing is for sure. this plugin should not create announcements.